packages/@aws-cdk/aws-dynamodb/test/integ.dynamodb.expected.json

```diff
@@ -20,8 +20,7 @@
           }
         ],
         "GlobalSecondaryIndexes": [],
-        "LocalSecondaryIndexes": [],
-        "Tags": []
+        "LocalSecondaryIndexes": []
       }
```

Comment:
Probably a separate PR but would be nice not to include Local and Global secondary indexes as well if they are empty

Reply:
Actually, that is what @rix0rrr commented previously. I agree with the idea. I will check how to achieve it.

Thanks for a quick review!